Client/Frontend/Browser/BrowserViewController/BrowserViewController+KeyCommands.swift

```diff
@@ -124,14 +143,26 @@ extension BrowserViewController {
             UIKeyCommand(input: "\\", modifierFlags: [.command, .shift], action: #selector(showTabTrayKeyCommand)), // Safari on macOS
             UIKeyCommand(input: "\t", modifierFlags: [.command, .alternate], action: #selector(showTabTrayKeyCommand), discoverabilityTitle: Strings.showTabTrayFromTabKeyCodeTitle)
         ]
+        
+        let bookmarkEditing = [
+            UIKeyCommand(input: "g", modifierFlags: [.command], action: #selector(showBookmarkCommand))
```

Comment:
This command should open a screen to add the currently selected tab as a bookmark, not to show a general bookmarks screen, sorry if i did not make it clear.
I believe `AddEditBookmarkTableViewController` is the VC we want to show

Reply:
Required change is made